code/Player/Player.Flashlight.cs

```diff
@@ -71,6 +71,9 @@ protected void DeleteFlashlight()
 		_viewLight = null;
 	}
 
+	[ConVar.Client( "ttt_visualizeflashlight" )]
+	public static bool VisualizeFlashlight { get; set; } = false;
```

Comment:
I think there are some `debug` preprocessors you can use for this, but, I think we'd just want to remove this.

Reply:
Yep, no worries. Just left the option there in case you wanted to include it.